src/woo_publications/publications/tests/test_documents_model.py

```diff
@@ -90,3 +97,68 @@ def test_status_change_with_none_existing_publication(self):
             )
 
             published_document.revoke()
+
+
+@override_settings(ALLOWED_HOSTS=["testserver", "host.docker.internal"])
+class TestRegisterInDocumentApi(VCRMixin, TestCase):
```

Comment:
Why couldn't you add this to the existing test case? Or otherwise move the existing test for this method to the new test case? There are now tests for this method in multiple places.